modules/migration-using-must-gather.adoc

```diff
@@ -4,11 +4,19 @@
 // * migration-toolkit-for-containers/troubleshooting-mtc
 
 [id="migration-using-must-gather_{context}"]
-= Using must-gather to collect data
 
-You must run the `must-gather` tool if you open a customer support case on the link:https://access.redhat.com[Red Hat Customer Portal] for the {mtc-full} ({mtc-short}).
 
-The `openshift-migration-must-gather-rhel8` image for {mtc-short} collects migration-specific logs and data that are not collected by the default `must-gather` image.
+
+= Using the must-gather tool
+
+You can collect logs, metrics, and information about {MTC-short} custom resources by using the `must-gather` tool.
+
+You can view metrics data with Prometheus, a component of the {product-title} monitoring stack.
```

Comment:
Not quite right. Prometheus also collects data. It's not like Graphana. See my [updated MTV must-gather section](https://access.redhat.com/documentation/en-us/migration_toolkit_for_virtualization/2.0/html-single/installing_and_using_the_migration_toolkit_for_virtualization/index#using-must-gather_mtv) to see how I wrote the second sentence.

Reply:
@apinnick 
That's what Derek W. explained to me.
But I copied the introductory part from MTV... Did you change it recently?

Reply:
yes